app/lib/efile/ny/it201.rb

```diff
@@ -413,10 +403,6 @@ def calculate_line_54
         [line_or_zero(:IT201_LINE_52) - line_or_zero(:IT201_LINE_53), 0].max
       end
 
-      def calculate_line_54b
-        (line_or_zero(:IT201_LINE_54A) * 0.0034).round
-      end
-
```

Comment:
this is the old MCTMT stuff, but line 54A was always zero so it wasn't really doing anything. if we need to implement this there are new values for 2023. will follow up separately.